server/src/main/java/com/google/fhir/gateway/FhirUtil.java

```diff
@@ -59,6 +61,27 @@ public static Bundle parseResponseToBundle(FhirContext fhirContext, HttpResponse
     return (Bundle) resource;
   }
 
+  public static IBaseResource createResourceFromRequest(
```

Comment:
This has simlar parsing logic as parseResponseToBundle.
It might be better to move the parsing to different method.

Reply:
The logic looks same as the methods being called in the two methods have the same name `jsonParser.parseResource`. However, the method signatures are different.
`parseResponseToBundle` uses the method with the signature
`IBaseResource parseResource(InputStream var1) throws ConfigurationException, DataFormatException;`
while `createResourceFromRequest` uses the method with the signature 
` IBaseResource parseResource(String var1) throws ConfigurationException, DataFormatException;`
While we can modify the latter one to use the one in the former one, I think there are only two lines of code that look like duplicates in the two methods mentioned and refactoring them might be over-refactoring since these two methods are provided out of the box and it makes sense to use them.

Reply:
sg